src/grisp_hw.erl

```diff
@@ -1,20 +1,97 @@
 % @doc GRiSP hardware access.
 -module(grisp_hw).
 
+-include("grisp_hw.hrl").
+
 % API
 -export([platform/0]).
+-export([read_eeprom/0]).
```

Comment:
I think we should prefix the EEPROM functions with `eeprom_` instead, so they're logically grouped.

Reply:
👍